corehq/apps/hqwebapp/templatetags/hq_shared_tags.py

```diff
@@ -726,13 +726,20 @@ class RequireJSMainNode(template.Node):
     def __init__(self, name, value):
         self.name = name
         self.value = value
+        self.origin = None
```

Comment:
I needed this to get the exception to display in my browser locally. Without it, an error about `origin` not being found supersedes the original exception.